packages/website/components/Playground/Playground.tsx

```diff
@@ -42,20 +32,12 @@ export default function Playground() {
   useEffect(() => {
     const query = qs.parse(window.location.search.slice(1));
     const decoded = tryDecode(query.code);
-    setCode(decoded || defaultCode);
+    setCode(decoded ?? defaultCode);
   }, []);
 
-  return (
-    <div className={styles.root}>
-      {code && <SandpackRenderer code={code} topbar={<PlaygroundTopBar />} />}
-    </div>
-  );
-}
-
-const defaultCode = `
-import { Button } from '@contentful/f36-components';
+  if (!code) {
+    return null;
+  }
```

Comment:
You might not need this check, `code` will always be a string

Reply:
If I remove it the whole thing explodes

![Screenshot 2022-01-25 at 16 48 24](https://user-images.githubusercontent.com/6597467/151010318-7e251650-c850-4e12-ab4d-641091296d6b.png)